### ISSUE TITLE
Docker in restart loop / not running

### ISSUE THREAD

**Issue by Snuffy2** [FlareSolverr/FlareSolverr#299]
### Environment

* **FlareSolverr version**: 2.2.1
* **Last working FlareSolverr version**: N/A
* **Operating system**: RPi Buster (RPi 4 4gb)
* **Are you using Docker**: [yes/no] yes
* **FlareSolverr User-Agent (see log traces or / endpoint)**:
* **Are you using a proxy or VPN?** [yes/no] VPN, but turned it off to install and test FlareSolverr
* **Are you using Captcha Solver:** [yes/no]
* **If using captcha solver, which one:**
* **URL to test this issue:**

### Description
Similar/same as #279 
Installed using the docker-compose. Running docker ps -a shows the container keeps restarting every ~15 seconds and isn't working.

### Logged Error Messages
`CONTAINER ID   IMAGE                                      COMMAND                  CREATED          STATUS                            PORTS    NAMES

0b505cd726c0   ghcr.io/flaresolverr/flaresolverr:latest   "/usr/bin/dumb-init …"   15 minutes ago   Restarting (133) 19 seconds ago             flaresolverr`

docker-compose.yml:
```
---
version: "2.1"
services:
  flaresolverr:
    # DockerHub mirror flaresolverr/flaresolverr:latest
    image: ghcr.io/flaresolverr/flaresolverr:latest
    container_name: flaresolverr
    environment:
      - LOG_LEVEL=debug
      - LOG_HTML=true
      #- LOG_LEVEL=${LOG_LEVEL:-info}
      #- LOG_HTML=${LOG_HTML:-false}
      - CAPTCHA_SOLVER=${CAPTCHA_SOLVER:-none}
      - TZ=Europe/London
    ports:
      - "${PORT:-8191}:8191"
    restart: unless-stopped
```

[flaresolverr.log](https://github.com/FlareSolverr/FlareSolverr/files/8010649/flaresolverr.log)

**Comment by ngosang** [comment]
Try these images:

ARM 32 =>
`ghcr.io/flaresolverr/flaresolverr:latest@sha256:8ce33db451708a17a075ffee8dc42cb7f7973be31ed9dc4c7222a3f035f05ee0`

ARM 64 =>
`ghcr.io/flaresolverr/flaresolverr:latest@sha256:c9cf15bf7e5d605a55db2ec2e3eb15c905189b47a7438da4b8b29ba1633808b2`

**Comment by Snuffy2** [comment]
Actually, after reading other issues further, I was able to solve it by upgrading libseccomp2 to greater than 2.4. I had previously setup the buster backports repo.
Looking at: `sudo apt-cache policy libseccomp2`
I was running 2.3.3-4 but 2.5.1-1 was available

I ran: `sudo apt install libseccomp2=2.5.1-1~bpo10+1`
Restarted the docker container and everything began working.

This is essentially the same as the manual upgrade of libseccomp2 described in #211 #249 etc.
I'd recommend adding the need to upgrade libseccomp2 to the docker instructions at least for ARM7/ARM32
